source/cloud/azure/azureml.md

```diff
@@ -228,11 +224,19 @@ command_job = command(
 
 # submit training job
 returned_job = ml_client.jobs.create_or_update(command_job)
+returned_job  # displays status and details page of the experiment
 ```
 
-After creating the job, go to [the "Experiments" page](https://ml.azure.com/experiments) to view logs, metrics, and outputs.
+After creating the job, click on the details page provided in the output of `returned_job`, or go
+to [the "Experiments" page](https://ml.azure.com/experiments) to view logs, metrics, and outputs.
 
-Next, try performing a sweep over a set of hyperparameters.
+```{note}
+For reference this job took ~7min while using `size="Standard_NC6s_v3"` in the `gpu_compute` creation
+```
+
+![Screenshot of job under the test_rapids_mlflow experiment](../../images/azureml_returned_job_completed.png)
+
+Next, we can perform a sweep over a set of hyperparameters.
```

Comment:
So I think we need to modify the parameters here, because this sweep creates 1000 jobs, I was able to complete ~5 in a span of 20 min, using `size="Standard_NC6s_v3"`. I'm not sure what's a more reasonable sweep but we should change this, any thoughts?

Reply:
I think something small is totally fine. The goal here is to show how to dispatch the jobs, not how to do a good sweep. If you think it's best to do a small number of sweeps as an illustration then just make a note to say "this is a small example, in reality you would do more".

Reply:
Yes, I think I'll modify this and test it very quickly so it's also easier for us to test if things finish as expected.